packages/components/src/components/Dropdown/index.tsx

```diff
@@ -169,9 +180,14 @@ const Dropdown = forwardRef(
         const [menuSize, setMenuSize] = useState<Coords>(undefined);
         const menuRef = useRef<HTMLUListElement>(null);
         const toggleRef = useRef<any>(null);
+        const onToggleChangeRef = useRef(onToggleChange);
         const MenuComponent = components?.DropdownMenu ?? Menu;
         const MenuItemComponent = components?.DropdownMenuItem ?? MenuItem;
 
+        useEffect(() => {
+            if (onToggleChangeRef?.current) onToggleChangeRef.current(toggled);
```

Comment:
Is this the right way how to call a function that was passed via props (I mean using the reference) ?

The effect is called even if `toggled` is not changed (it's called when onToggleChange changes it's value as well).